docs/core-concepts/authenticating-with-api-keys.mdx

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: 'Authenticating with API Keys'
+---
+
+# Authenticating with API Keys
+
+The Regression Games platform provides the ability to generate API keys, which can be used
+to authenticate against our API. This is the recommended way to authenticate against our API
+when connecting external systems to Regression Games.
+
+You can find the **API Keys** manager at the bottom of your [Regression Games account settings](https://play.regression.gg/account).
```

Comment:
When I clicked on the link my brain looked for API Keys Manager, maybe calling it API Keys section will be better.

Reply:
changed to "section"